metricbeat/docker-compose.yml

```diff
@@ -46,13 +47,26 @@ services:
     volumes:
       - ${PWD}/..:/go/src/github.com/elastic/beats/
       - /var/run/docker.sock:/var/run/docker.sock
+      - datavolume:/etc/ceph
+      - datavolume1:/var/run/ceph
     command: make
     entrypoint: /go/src/github.com/elastic/beats/metricbeat/docker-entrypoint.sh
 
   # Modules
   apache:
     build: ${PWD}/module/apache/_meta
 
+  ceph:
+    image: ceph/demo
+    container_name: metricbeatceph
```

Comment:
Same as above :)
"I'll study the rest api but I think that we can remove all this and don't use command any more."